cmd/infra/powervs/create.go

```diff
@@ -1159,13 +1165,40 @@ func (infra *Infra) setupPowerVSDHCP(ctx context.Context, options *CreateInfraOp
 	return nil
 }
 
+// isRetryableError validates err contains possible retryable error keywords like timeoutErrorKeywords, if yes, the action can be retried and increase the retryCount, if not return the error.
+// if retryCount exceeds retryLimit, a retry limit exceeded error will be thrown.
+func isRetryableError(err error, retryableErrorKeywords []string, retryCount *int) (bool, error) {
```

Comment:
do we need bool here? I see always we returned `false, err` or `true, nil`, I think we can remove bool return.